webserver/websocket/src/main/java/io/helidon/webserver/websocket/WsConnection.java

```diff
@@ -85,14 +88,16 @@ private WsConnection(ConnectionContext ctx,
      * @param upgradeHeaders headers for
      * @param wsKey          ws key
      * @param wsRoute        route to use
+     * @param wsConfig       websocket config
      * @return a new connection
      */
     public static WsConnection create(ConnectionContext ctx,
                                       HttpPrologue prologue,
                                       Headers upgradeHeaders,
                                       String wsKey,
-                                      WsRoute wsRoute) {
-        return new WsConnection(ctx, prologue, upgradeHeaders, wsKey, wsRoute);
+                                      WsRoute wsRoute,
+                                      WsConfig wsConfig) {
+        return new WsConnection(ctx, prologue, upgradeHeaders, wsKey, wsRoute, wsConfig);
```

Comment:
Public API change in dot release, can we keep the old version and deprecate it?

Reply:
Good catch, let me see how we can do this differently.